tools/code-analyzer/README.md

```diff
@@ -8,22 +8,21 @@
 
 Currently there are just 2 commands:
 
-1. `analyzer`. Analyzer serves 2 roles currently, as a linter for PRs to check if introduced hook/template/db changes have associated changelog entries and also to provide file output of changes between
-WooCommerce versions for the purpose of automating release processes (such as generating release posts.)
+1. `lint`. Analyzer is used as a linter for PRs to check if hook/template/db changes were introduced. It produces output either directly on CI or via GH actions `set-output`.
 
 Here is an example `analyzer` command:
 
-`./bin/dev analyzer release/6.8 "6.8.0" -b=release/6.7`
+`pnpm run analyzer -- lint "release/6.8" "6.8.0" -b=release/6.7`
```

Comment:
This command fails for me. I have the `release/6.7` branch in my local git, so not sure whats happening.
```
GitError: error: pathspec '=release/6.7' did not match any file(s) known to git
```

Reply:
This is a mess up from me in the syntax of the command. it should be `-b release/6.7` I'll fix

Reply:
Resolved in https://github.com/woocommerce/woocommerce/pull/34600/commits/691fca5efc62588870acdd13894b6d9ffc858ed6